PopulateTools/gobierto
# Budget description localization

Connects to #74

### What does this PR do?

This PR introduces the concept of site locale, as a configuration parameter. This is the minimum we need for the moment (no UI yet).

It also sets a controller filter to change the locale based on the site configuration.

And, of course, parameterizes budget categories description based on the locale, to show the correct description for the current locale.

### How should this be manually tested?

You should set a locale in the site configuration:

```
s = Site.first
s.configuration.locale = :ca`
s.save!
```

And then go to the public URL of the site, and you should see the budgets and budget descriptions in catalan.
